src/engine/DebugOverlays.tscn

```diff
@@ -209,12 +209,12 @@ __meta__ = {
 "_editor_description_": "PLACEHOLDER"
 }
 
-[connection signal="Closed" from="PerformanceMetrics" to="." method="OnPerformanceMetricsToggled"]
+[connection signal="Cancelled" from="PerformanceMetrics" to="." method="OnPerformanceMetricsToggled"]
```

Comment:
I looked at the code for PerformanceMetrics, and it really looks like to me like it only cares about if the popup becomes hidden, it doesn't care about anything else. So this is exactly my point that the most natural sounding signal name should be used in each case. So for this and probably a lot of other cases, the cancelled signal should be switched to be the Closed one (while still keeping the cancelled signal for cases that make most sense). Due to Easter I'm running out of time this week, so I won't be checking this until next week. Then I'll experiment and help finish this PR myself if I still see potential ways the system can be made more clear (after all this is a pretty core system that all parts of the GUI will use for years to come).

Reply:
Sure, that would be great. Wish you a good week!